src/components/common/Checkbox/Checkbox.css

```diff
@@ -11,4 +11,8 @@
 .checkbox-label {
   margin: 0 4px;
   cursor: pointer;
+}
+
```

Comment:
```suggestion

  .dark .checkbox-wrapper {
    background-color: #333;
  }

```
On the PeopleReport page.
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/8aacee93-ba92-4c36-93c6-4a9aa4dbabea)